src/app/app.component.ts

```diff
@@ -1126,6 +1126,15 @@ export class AppComponent implements OnInit {
     }
   }
 
+  toggleDisplayOrder = () => {
+    // Toggle display order to change classes for collection order or reading order
```

Comment:
These lines can be replaced with `this.isShowReadingOrder = !this.isShowReadingOrder`